File: Tarea4.js
// MÉTODO DE ORDENAMIENTO BURBUJA (JavaScript)
// ================================
let numeros = [64, 34, 25, 12, 22, 11, 90];

console.log("Lista original:", numeros);

let n = numeros.length;

// Algoritmo burbuja
for (let i = 0; i < n - 1; i++) {
    for (let j = 0; j < n - 1 - i; j++) {
        if (numeros[j] > numeros[j + 1]) {
            let temp = numeros[j];
            numeros[j] = numeros[j + 1];
            numeros[j + 1] = temp;
        }
    }
}

console.log("Lista ordenada:", numeros);

File: Tarea4.cpp
// MÉTODO DE ORDENAMIENTO BURBUJA (C++)
#include <iostream>
using namespace std;

int main() {
    int numeros[] = {64, 34, 25, 12, 22, 11, 90};
    int n = sizeof(numeros) / sizeof(numeros[0]);

    cout << "Lista original: ";
    for (int i = 0; i < n; i++) {
        cout << numeros[i] << " ";
    }
    cout << endl;

    // Algoritmo burbuja
    for (int i = 0; i < n - 1; i++) {
        for (int j = 0; j < n - 1 - i; j++) {
            if (numeros[j] > numeros[j + 1]) {
                int temp = numeros[j];
                numeros[j] = numeros[j + 1];
                numeros[j + 1] = temp;
            }
        }
    }

    cout << "Lista ordenada: ";
    for (int i = 0; i < n; i++) {
        cout << numeros[i] << " ";
    }
    cout << endl;

    return 0;
}